src/aiq/runtime/session.py

```diff
@@ -103,7 +103,7 @@ async def session(self,
                 self._context_state.user_input_callback.reset(token_user_input)
 
     @asynccontextmanager
-    async def run(self, message):
+    async def run(self, message, entry_fn: str | None = None):
```

Comment:
We dont want to add this here. It adds an extra burden to every `run` call. There is another method in the workflow builder that will create a new workflow from a different entrypoint. Use that instead. Look at the fastapi front end for more info